src/main/pages/che-7/hands-on-guidance/proc_adding-the-vs-code-extension-using-the-che-plugins-panel.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// adding-che-plug-in-registry-vs-code-extension-to-a-workspace
+
+// This module can be included from assemblies using the following include statement:
+// include::<path>/proc_adding-the-vs-code-extension-using-the-che-plugins-panel.adoc[leveloffset=+1]
+
+// The file name and the ID are based on the module title. For example:
+// * file name: proc_doing-procedure-a.adoc
+// * ID: [id='proc_doing-procedure-a_{context}']
+// * Title: = Doing procedure A
+//
+// The ID is used as an anchor for linking to the module. Avoid changing
+// it after the module has been published to ensure existing links are not
+// broken.
+//
+// The `context` attribute enables module reuse. Every module's ID includes
+// {context}, which ensures that the module has a unique ID even if it is
+// reused multiple times in a guide.
+//
+// Start the title with a verb, such as Creating or Create. See also
+// _Wording of headings_ in _The IBM Style Guide_.
+[id="adding-the-vs-code-extension-using-the-che-plugins-panel_{context}"]
+= Adding the VS Code extension using the Che Plugins panel
+
+To add the VS Code extension using the *Che Plugin* panel, follow the procedure below.
+
+[discrete]
+== Procedure
+
+. Open the *Che Plugin* panel.
+
+. Change the current registry to the registry in which the VS Code extension was added.
+
+. In the search bar, click the *Menu* button and then click *Change Registry* to choose the registry from the list. If the required registry is not in the list, add it using the `Add Registry` menu option. The registry link should point to the `plug-ins` segment of the registry. See an example: `https://my-registry.com/v3/plugins/index.json`.
++
+image::extensibility/adding_the_vs_code_extension_changeRegistry.png[]
+
+. Search for the required plug-in using the filter and then click the *Install* button.
```

Comment:
```suggestion
. Search for the required plug-in using the filter and then click the btn:[Install] button.
```

Reply:
Well, the result of "btn" and "bold" markup looks exactly the same after rendering but I changed this to the button